### DISCUSSION TITLE
Date Time format of nodes

### DISCUSSION THREAD

**Discussion by Hari-quartic** [FreeOpcUa/opcua-asyncio#514] [Q&A]
Hello Asyncio team,

I am getting error as "_[OPCUAConnector:Error]An error occurred in parsing data for Node ns=2;i=4 with EdgeConnectorId 744. Details: The UTC time represented when the offset is applied must be between year 0 and 10,000._" Can someone pls advise whats the format of datetime server-minimal.py streams.

**Comment by swamper123** [comment]
> [OPCUAConnector:Error]An error occurred in parsing data for Node ns=2;i=4 with EdgeConnectorId 744. Details: The UTC time represented when the offset is applied must be between year 0 and 10,000." 

What kind of client are you using? 

UaExpert is working fine:
![grafik](https://user-images.githubusercontent.com/24408657/112635364-0354c080-8e3c-11eb-81ae-b8cda4951039.png)

ServerTimestamp may have to be set, because this is the default start time of OPC UA.

**Reply by Hari-quartic** [reply]
Thanks for your answer. Client is using _OPCUA Foundation .NET SDK_ library.  
Is there a method to change the format of timestamp ?

**Reply by swamper123** [reply]
I am not familiar with that Client, so I can neither confirm nor deny it, whose false it is. 
Tbh I don't understand what the client is complaining about (what format it expects and what we may do wrong atm)?
